src/backends/cargo.rs

```diff
@@ -161,6 +164,7 @@ fn extract_packages(contents: &str) -> Result<BTreeMap<String, CargoOptions>> {
                     all_features,
                     no_default_features,
                     features,
+                    locked: false, //cargo does not track if the install happened with --locked
```

Comment:
This could be misleading as we don't know if it's locked but are instead saying it wasn't locked. How about using an `Option<bool>` instead?

Reply:
Good point. I guess it would be consistent with flatpak_default_systemwide if I then also add it to the config with a default false?

Reply:
Oh wait no I think I misunderstood your comment. You're bang on. Yep a `cargo_default_locked` would be make sense as to what to do if `locked == None`